contents/links.yml

```diff
@@ -83,7 +83,7 @@ externalLinks:
     ariaLabel: "Agenzia per l'Italia Digitale (link esterno)"
   a11y:
     label: Dichiarazione di accessibilità
-    linkTo: https://form.agid.gov.it/prossimapa.gov.it
+    linkTo: https://form.agid.gov.it/padigitale2026.gov.it
```

Comment:
This should link to the actual accessibility page.

Reply:
nope, it is a temporary placeholder, this link is actually hidden in UI